LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
-
-Copyright (c) 2017-present stylelint
```

Comment:
[question] Is the copyright year change (2017 -> 2018) no problem?

https://github.com/stylelint/.github/blob/cdaa743eab2551597b776fcc62661a3e79872650/LICENSE#L3

Reply:
No problem. If it is, we can change org year to 2015 when we first started the org.

Reply:
I see. Thanks.